wavenet_ops.py

```diff
@@ -32,6 +34,29 @@ def causal_conv(value, filter_, dilation, name='causal_conv'):
             restored = tf.nn.conv1d(padded, filter_, stride=1, padding='SAME')
         # Remove excess elements at the end
         result = tf.slice(restored,
-                         [0, 0, 0],
-                         [-1, tf.shape(value)[1], -1])
+                          [0, 0, 0],
+                          [-1, tf.shape(value)[1], -1])
         return result
+
+
+def mu_law_encode(audio, quantization_channels):
+    '''Quantizes waveform amplitudes.'''
+    with tf.name_scope('encode'):
+        mu = quantization_channels - 1
+        # Perform mu-law companding transformation (ITU-T, 1988).
+        magnitude = tf.log(1 + mu * tf.abs(audio)) / tf.log(1. + mu)
+        signal = tf.sign(audio) * magnitude
+        # Quantize signal to the specified number of levels
```

Comment:
Besides that (which we can ignore if you feel lazy about it), it LGTM. There's some docstrings missing, but we can add them later.